--- MD_Parola/src/MD_Parola.h	2018-04-26 00:04:03.000000000 +0200
+++ ./MD_Parola/src/MD_Parola.h	2018-05-15 18:09:44.878233076 +0200
@@ -926,7 +926,7 @@
   /***
     *  Structure for list of user defined characters substitutions.
   */
-  typedef struct charDef_t
+  struct charDef_t
   {
     uint8_t   code;   ///< the ASCII code for the user defined character
     uint8_t   *data;  ///< user supplied data
